apps/kaizen/models.py

```
@@ -1,3 +1,56 @@
+# SubTask model
+class SubTask(BaseModel):
+    task = models.ForeignKey(Task, on_delete=models.CASCADE, related_name="subtasks")
+    title = models.CharField(max_length=255)
+    completed = models.BooleanField(default=False)
+    completed_time = models.DateTimeField(blank=True, null=True)
+    is_all_day = models.BooleanField(default=False)
+    start_date = models.DateTimeField()
+    time_zone = models.CharField(max_length=50, blank=True, null=True)
+    sort_order = models.IntegerField()
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)